source/test/TestApp/PlayFabQoSTest.cpp

```diff
@@ -2,10 +2,11 @@
 
 #include "TestAppPch.h"
 
-#if defined(PLAYFAB_PLATFORM_WINDOWS) || defined(PLAYFAB_PLATFORM_XBOX)
+//#if defined(PLAYFAB_PLATFORM_WINDOWS) || defined(PLAYFAB_PLATFORM_XBOX)
```

Comment:
You can't blanket remove this without thoroughly testing on every other platform.
At the moment, that's an intense process due to the way the private templates lock to specific core commits (private templates don't test latest core when you make changes to core - yet).